### ISSUE TITLE
retif missing semicolon

### ISSUE THREAD

**Issue by mgrip** [prettier/plugin-php#42]
input:
```php
$test ? doSomething() : doSomethingElse();
```
output:
```php
$test ? doSomething() : doSomethingElse()
```

**Comment by czosel** [comment]
I wasn't able to reproduce this when i added your exact line to the tests - maybe it depends on the context this line is found in?

**Comment by mgrip** [comment]
This is still happening for me - I'll open a new MR

**Comment by mgrip** [comment]
fixed with #64
